library.properties

```diff
@@ -1,5 +1,5 @@
 name=DualVNH5019MotorShieldMod3
-version=2.0.0.dualshilds
+version=2.0.0-dualshilds
```

Comment:
`version=3.0.0-dualshilds`

Reply:
OK, I wasn't sure how you would want to handle the version. I have pushed a commit that sets it to `3.0.0-dualshilds`. If you prefer something else let me know and I'll change it.

Reply:
Truthfully, I'm not sure how I want to handle the version numbers, I have thoughts on rewriting this library using the parent library as a dependency and just make the functions here pass to the parent library. In my mind that would allow dropping the postfix I'm using here.

Reply:
I had a fork of a library where I just wanted to add a few features (that had not been accepted upstream), and otherwise to follow the work on the parent. I wanted my versioning to still indicate which parent version it was based on. So I appended my versioning to the parent versioning using the semver pre-release syntax (e.g. `1.1.0-mod.1.1.3`. It was an abuse of the pre-release syntax, but the best solution I could come up with while remaining semver compliant.